browser/js/global.js

```diff
@@ -228,7 +228,13 @@ window.app = {
 
 		// @property retina: Boolean
 		// `true` for browsers on a high-resolution "retina" screen.
-		retina: (window.devicePixelRatio || (window.screen.deviceXDPI / window.screen.logicalXDPI)) > 1
+		retina: (window.devicePixelRatio || (window.screen.deviceXDPI / window.screen.logicalXDPI)) > 1,
+
+		getLang: function () {
+			return (navigator.languages && navigator.languages.length ? navigator.languages[0] :
+				navigator.language || navigator.userLanguage ||
+				navigator.browserLanguage || navigator.systemLanguage);
```

Comment:
Is this supposed to be 
`navigator.languages && navigator.languages.length ? navigator.languages[0] : (navigator.language || navigator.userLanguage ||
				navigator.browserLanguage || navigator.systemLanguage)` 
or 
`(navigator.languages && navigator.languages.length ? navigator.languages[0] :
				navigator.language) || navigator.userLanguage ||
				navigator.browserLanguage || navigator.systemLanguage` ?

Perhaps adding `()` would make it more readable.